### PR TITLE
Add prow-dashboard to grafana of prow-monitoring

### PR DESCRIPTION
**Pull request by hongkailiu**
https://github.com/kubernetes/test-infra/issues/12854

### CODE REVIEW

File: prow/monitoring/grafana_deployment.yaml [hunk 1]
```diff
@@ -40,6 +40,9 @@ spec:
         - mountPath: /etc/grafana
           name: grafana-config
           readOnly: false
+        - mountPath: /grafana-dashboard-definitions/0/prow
```
Comment (cjwagner) [inline prow/monitoring/grafana_deployment.yaml]: Why does this path have a `prow` suffix?

Comment (stevekuznetsov) [reply prow/monitoring/grafana_deployment.yaml]: IIRC it's the name of the group of dashboards?

**Comment by stevekuznetsov** [comment]
```
bazel run //prow/monitoring:production.apply
Starting local Bazel server and connecting to it...
INFO: Analyzed target //prow/monitoring:production.apply (31 packages loaded, 447 targets configured).
INFO: Found 1 target...
INFO: Deleting stale sandbox base /home/skuznets/.cache/bazel/_bazel_skuznets/fe650a89676a24b8d48936a0c5c19473/sandbox
Target //prow/monitoring:production.apply up-to-date:
  bazel-bin/prow/monitoring/production.apply
INFO: Elapsed time: 6.482s, Critical Path: 0.74s
INFO: 11 processes: 11 linux-sandbox.
INFO: Build completed successfully, 12 total actions
INFO: Build completed successfully, 12 total actions
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
namespace/prow-monitoring unchanged
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
clusterrolebinding.rbac.authorization.k8s.io/prow-prometheus-operator unchanged
clusterrole.rbac.authorization.k8s.io/prow-prometheus-operator configured
serviceaccount/prometheus-operator unchanged
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
deployment.apps/prometheus-operator configured
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
servicemonitor.monitoring.coreos.com/prometheus unchanged
servicemonitor.monitoring.coreos.com/alertmanager unchanged
servicemonitor.monitoring.coreos.com/grafana unchanged
servicemonitor.monitoring.coreos.com/deck unchanged
servicemonitor.monitoring.coreos.com/ghproxy unchanged
servicemonitor.monitoring.coreos.com/hook unchanged
servicemonitor.monitoring.coreos.com/plank unchanged
servicemonitor.monitoring.coreos.com/sinker unchanged
servicemonitor.monitoring.coreos.com/tide unchanged
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
serviceaccount/prometheus-prow unchanged
clusterrolebinding.rbac.authorization.k8s.io/prometheus-prow unchanged
clusterrole.rbac.authorization.k8s.io/prometheus-prow configured
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
prometheus.monitoring.coreos.com/prow unchanged
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
service/prometheus unchanged
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
serviceaccount/grafana unchanged
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
service/grafana unchanged
ingress.extensions/grafana configured
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
configmap/grafana-datasources unchanged
configmap/grafana-config unchanged
configmap/grafana-dashboards unchanged
configmap/grafana-dashboard-prow created
$ /usr/bin/kubectl --cluster=gke_k8s-prow_us-central1-f_prow --context= --user= apply -f -
deployment.apps/grafana configured
```

**Comment by stevekuznetsov** [comment]
http://monitoring.prow.k8s.io/d/970b051d3adfd62eb592154c5ce80377/prow-dashboard?orgId=1

**Pull request merged**